app/models/alchemy/content/factory.rb

```diff
@@ -19,7 +19,8 @@ def build(element, essence_hash)
         if (description = content_description(element, essence_hash)).blank?
           raise ContentDefinitionError, "No description found in elements.yml for #{essence_hash.inspect} and #{element.inspect}"
         else
-          new(name: description['name'], element_id: element.id)
+          translate = description['translate'] == false ? false : true
+          new(name: description['name'], element_id: element.id, translate: translate)
```

Comment:
"I did place a default of true in the db and wrote a rake task to add true to the current contents."

^^^ wouldn't be necessary if you had a column that you only needed to take action on when it was present and true.

In other words, absence of a value or explictly set to 'false' is ignorable.  explicitly set to 'true'  would be the only time you'd take action.

What do you think?